modules/deepmerge/variables.tf

```diff
@@ -5,12 +5,24 @@ variable "maps" {
     condition     = try(tonumber(var.maps), tobool(var.maps), tostring(var.maps), tomap(var.maps), null) == null
     error_message = "The `maps` variable must be a list of maps and/or objects. The provided value is not a list."
   }
-  validation {
-    condition = ! can(index([
-      for mp in var.maps :
-      try(tolist(mp), toset(mp), tonumber(mp), tobool(mp), tostring(mp), null) == null
-      ]
-    , false))
-    error_message = "The `maps` variable must be a list of maps and/or objects. Not all elements meet this requirement."
-  }
+  #  validation {
+  #    condition = ! can(index([
+  #      for mp in var.maps :
+  #      try(tolist(mp), toset(mp), tonumber(mp), tobool(mp), tostring(mp), null) == null
+  #      ]
+  #    , false))
+  #    error_message = "The `maps` variable must be a list of maps and/or objects. Not all elements meet this requirement."
+  #  }
+}
+
+variable "append_list" {
```

Comment:
These should be suffixed with `_enabled` ideally (might need rewording). https://docs.cloudposse.com/reference/best-practices/terraform-best-practices/#use-feature-flags-to-enabledisable-functionality